src/modules/LanItems.rb

```diff
@@ -1580,19 +1565,37 @@ def BuildLanOverview
           end
           @current = curr
         end
-        LanItems.Items[key]["table_descr"] = {
+        # FIXME: - find how to enable bellow code
+#        LanItems.Items[key]["table_descr"] = {
+#          "rich_descr"  => rich,
+#          "table_descr" => [descr, DeviceProtocol(ifcfg_conf), ifcfg_name, note]
+#        }
+        overviews[iface.name]["table_descr"] = {
           "rich_descr"  => rich,
-          "table_descr" => [descr, DeviceProtocol(ifcfg_conf), ifcfg_name, note]
+          "table_descr" => [descr, DeviceProtocol(iface.config), iface.name, note]
         }
       end
-      [Summary.DevicesList(overview), links]
+
+      {
+        overview: overview,
+        overviews: overviews,
+        links: links
+      }
```

Comment:
This should be presenter for sure.

Reply:
definitely agree ... the future is bright ;-)